HOW/_requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+HOW: <path in HOW tree>
+Source: <path to process used to create this HOW>
+Created: <YYYY-MM-DD>
+Version: <version-type(semantic/index)/version-number>
+Status: <initial status from process>
+---
+
+## Title
+The title of the standard being defined
+
+## Author(s)
+
+list of authors' names and optionally, email addresses
```

Comment:
I would personally prefer to go by my GitHub ID

Reply:
Right, it makes sense to broaden that.